src/utils/coords-parser.ts

```diff
@@ -0,0 +1,62 @@
+export class DecimalCoordinates {
```

Comment:
I have some doubts about use of this class, please specify need in it

Reply:
It is just for typing in function fromEuringToDecimal. I can create Interface instead of class and return object from this function ( eg. { longitude: data, latitude: data }). Would it be Ok?

Reply:
agree. would be good!

Reply:
We are usually use interfaces then we would like to describe 'behavior'. I think you can just create type like 
`type DecimalCoordinates = {
  latitude: string;
  longitude: string;
}`
but its only my opinion))

Reply:
Thank, it will be better.=)

Reply:
Fixed.